pkg/volume/rbd/rbd_util.go

```diff
@@ -407,6 +407,51 @@ func (util *RBDUtil) CreateImage(p *rbdVolumeProvisioner) (r *v1.RBDVolumeSource
 	}, sz, nil
 }
 
+func (util *RBDUtil) CreatePersistentVolumeImage(p *rbdVolumeProvisioner) (r *v1.RBDPersistentVolumeSource, size int, err error) {
```

Comment:
should be able to change the existing CreateImage method... there are no other callers

Reply:
done